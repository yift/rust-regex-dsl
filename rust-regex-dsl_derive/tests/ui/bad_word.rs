pub use rust_regex_dsl::regex_dsl;

fn main() {
    let regex = regex_dsl! {
        nop
    };
    println!("{}", regex);
}