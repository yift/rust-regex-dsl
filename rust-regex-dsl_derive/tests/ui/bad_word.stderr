error: Unknown word: nop
 --> tests/ui/bad_word.rs:5:9
  |
5 |         nop
  |         ^^^
